common/src/benchmarks.c

```diff
@@ -47,12 +61,25 @@ static void __attribute__((noinline)) do_loads_cached(void) {
     buf[i] = acc;  // inhibit optimization
   }
   int end_time = perf_get_mcycle();
+
   int total_iters = 1024 * 1024;
-  printf("Val:%d  Cycles: %d   Cycles/load: %d\n\n\n", acc,
+  printf("Val:%d  Cycles: %d   Cycles/load: %d\n", acc,
          end_time - start_time, (end_time - start_time) / total_iters);
+
+  get_csr_metrics(&acc_cnt, &refill_cnt, &stall_cnt);
```

Comment:
Can we introduce a macro or helper function to avoid copy-pasting this into all the benchmarks?

Reply:
Sure, I can split it into two macros: one for declaring variables and getting the numbers before a test/benchmark, and one that gets the numbers after running a test/benchmark and prints them.

Reply:
Should be done now.